docs/rules/boolean-prop-naming.md

```diff
@@ -105,7 +106,7 @@ message: 'It is better if your prop ({{ propName }}) matches this pattern: ({{ p
 
 And the failure would look like so:
 
-```
+```plaintext
```

Comment:
why is this needed? triple backtick sections don't *have* to declare a language

Reply:
seems like it's probably console output, so maybe `sh` then?

Reply:
`sh` doesn't make sense for this because it will try to highlight shell commands in this English sentence.